src/connection.js

```diff
@@ -290,6 +290,22 @@ class Connection {
 		}
 	}
 
+	/**
+	 * Normalisation of the namespace to a value that is compatible with the OpenEO specs.
+	 *
+	 * This is required to support UDP that are shared as public. These can only be executed with providing the full URL
+	 * (e.g. https://<backend>/processes/u:<user>/<udp_id>) as the namespace value in the processing graph. For other
+	 * parts of the API (such as the listing of the processes, only the name of the namespace is required (u:<user>).
+	 *
+	 * This function will extract the short name of the namespace from a shareable URL.
+	 * @param namespace
+	 * @returns {*}
+	 */
+	normalizeNamespace(namespace) {
+		const matches = namespace.match( /^http[s]?:\/\/.*\/processes\/(u:[\d\w]*)\/?/);
```

Comment:
The `u:` is VITO specific and not standardized, thus changing the regexp to the official proposal in https://github.com/Open-EO/openeo-api/pull/348 - this still matches the `u:xxx` namespaces

```suggestion
		const matches = namespace.match( /^https?:\/\/.*\/processes\/(@?[\w\-\.~:]+)\/?/i);
```